LocalMultiplayerAgent/Globals.cs

```diff
@@ -20,6 +20,8 @@ public static class Globals
         // Value updated from settings.json
         public static SessionConfig SessionConfig = null;
 
+        public static bool CreateDeployment = false;
```

Comment:
"Deployment" is an internal MPS name. "Build" is the name we expose to customers via our public APIs. I believe we should keep it consistent.